.github/workflows/rust.yml

```diff
@@ -0,0 +1,22 @@
+name: Rust
```

Comment:
Potentially a bad name but otherwise nice!

Reply:
Auto-generated... I'll change it if I add something else to the script (Y)